src/explore-education-statistics-admin/src/pages/publication/PublicationMethodologyPage.tsx

```diff
@@ -0,0 +1,477 @@
+        <>
+          <Link
+            to={generatePath<PublicationRouteParams>(
+              externalMethodologyEditRoute.path,
+              {
+                publicationId,
+              },
+            )}
+          >
+            Add external methodology
+          </Link>
+          <p>
+            This is a link to an existing methodology that is hosted externally
+          </p>
+        </>
+      )}
+
+      {permissions.canAdoptMethodologies && (
+        <>
+          <Link
+            to={generatePath<PublicationRouteParams>(
+              methodologyAdoptRoute.path,
+              {
+                publicationId,
+              },
+            )}
+          >
+            Adopt an existing methodology
+          </Link>
+          <p>This is a methodology that is owned by another publication</p>
+        </>
+      )}
+
+      {amendMethodologyId && (
+        <ModalConfirm
+          open
+          title="Confirm you want to amend this published methodology"
+          onConfirm={async () => {
+            const amendment = await methodologyService.createMethodologyAmendment(
+              amendMethodologyId,
+            );
+            history.push(
+              generatePath<MethodologyRouteParams>(
+                methodologySummaryRoute.path,
+                {
+                  methodologyId: amendment.id,
+                },
+              ),
+            );
+          }}
+          onExit={() => setAmendMethodologyId(undefined)}
+          onCancel={() => setAmendMethodologyId(undefined)}
+        >
+          <p>
+            Please note, any changes made to this published methodology must be
+            approved before updates can be published.
+          </p>
+        </ModalConfirm>
+      )}
+
+      {deleteMethodologyDetails && (
+        <ModalConfirm
+          open
+          title={
+            deleteMethodologyDetails.amendment
+              ? 'Confirm you want to cancel this amended methodology'
+              : 'Confirm you want to delete this methodology'
+          }
+          onConfirm={async () => {
+            await methodologyService.deleteMethodology(
+              deleteMethodologyDetails?.methodologyId,
+            );
+            setDeleteMethodologyDetails(undefined);
+            onReload();
+          }}
+          onCancel={() => setDeleteMethodologyDetails(undefined)}
+          onExit={() => setDeleteMethodologyDetails(undefined)}
+        >
+          <p>
+            {deleteMethodologyDetails.amendment ? (
+              <>
+                By cancelling the amendment you will lose any changes made, and
+                the original methodology will remain unchanged.
+              </>
+            ) : (
+              <>By deleting this methodology you will lose any changes made.</>
+            )}
+          </p>
+        </ModalConfirm>
+      )}
+
+      {dropMethodologyId && (
+        <ModalConfirm
+          open
+          title="Remove methodology"
```

Comment:
Similarly to above, if we change the terminology from 'remove' to something like 'unlink/detach', we should also change this modal accordingly.

Reply:
I've update the text to match the changes described above.